### DISCUSSION TITLE
I select 'em, jellyfin removes 'em

### DISCUSSION THREAD

**Discussion by jimlynnjulian** [jellyfin/jellyfin#8051] [General] [answered]
There is a serious issue with jellyfin replacing user selected imagery. 
I spend a great deal of time selecting poster (Primary) images and backdrop  images . Hours, days, or weeks later, they've been replaced by software selected images.
The answer is simple. Put a lock on it. No image edits without explicit user permissions after initialization.
Does jellyfin have whatever permissions I have when I log on?
If I log on as Admin, does je;llyfin have Admin permissions?
Any ideas on how to lock jellyfin out of my image collection?

**Comment by oddstr13** [comment]
Does not locking metadata also lock the images?
«Lock this item to prevent future changes» under «Edit metadata»

**Reply by jimlynnjulian** [reply]
I'm working with 15TB of data in 10 external hard drives. Locking/unlockiing the data takes a long time. A 5TB drive takes about an hour.
This may be due to not using nfo files. I've changed that, but, since the lock process is so time consuming, I need everything unlocked to troubleshoot the system.

**Comment by jimlynnjulian** [comment, marked as answer]
Options:
1. Use robocopy to mirror a drive with image and info files includded. Periodically reverse the copy to restore original image set.
2.  Use Windows security tools to lock the drive.
3. Use attrib to make a drive read only.
